readthedocs/templates/projects/project_import.html

```diff
@@ -209,6 +209,26 @@ <h1>{% trans "Import a Repository" %}</h1>
         </div>
       {% endblock %}
 
+      {# accounts is a dictionary with the provider as key and a list of connected accounts of that provider #}
+      {% get_social_accounts user as accounts %}
+      {% if accounts %}
+        <h3>{% trans "Filter by Own Repositories" %}</h3>
+        {% for type, list in accounts.items %}
+          {% for account in list %}
+            <li
+              class="remote-org"
```

Comment:
Use a proper CSS class for this like `social-account` maybe?

Reply:
I will need help here, since the `remote-user` doesn't exist and the `remote-org` is defined in the `import.less` file. I'd say that they should be almost the same. So, I will need some guideance to not repeat the whole thing.